e2e-tests/cypress/e2e/client-display/client-display-skill-paging_spec.js

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright 2020 SkillTree
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+describe('Client Display Skills Navigation', () => {
+
+    it('navigate between skills', () => {
```

Comment:
this test looks great! please add another tests that starts on a Skill page and does similar type of prev/next navigation

Reply:
Added more tests to the spec